modules/singlePage.js

```diff
@@ -0,0 +1,41 @@
+// spa.js
+export default function initializeSPA() {
```

Comment:
as suggested previously you are required to use the ES6 syntax, please use arrow functions for your initialize SPA function, please fix this 👍 👍